src/Processor/OrderPaymentProcessor.php

```diff
@@ -21,7 +21,7 @@ final class OrderPaymentProcessor implements OrderProcessorInterface
 
     public function __construct(
         OrderProcessorInterface $baseOrderPaymentProcessor,
-        FactoryInterface $stateMachineFactory
+        FactoryInterface $stateMachineFactory,
```

Comment:
I believe, we should handle `sylius/state-machine-abstraction` case here as well.
I think we should also address the `sylius/state-machine-abstraction` case here.

Reply:
I've added SyliusStateMachineAbstractionBundle in places where stateMachineFactory was used